app/views/welcome/_next-event.html.erb

```diff
@@ -18,16 +18,14 @@
     <% else %>
       <p>
         There are currently no presentations for this event.
-        <%= link_to 'Sign up', new_presentation_path %>
+        <%= render 'next-event-links' %>
```

Comment:
I think this is fine for now but maybe a hint for the user that they have to sign in to create one in the future. There are currently no presentations for this event... sign in. Feels weird. I like There are currently no presentations for this event... sign in to create one. Either is fine really.